Podfile.lock

```diff
@@ -1,8 +1,8 @@
 PODS:
```

Comment:
Generally I like to track the .lock file to define which exact versions this is ensured to work with. In this case, since there are only dependencies for the tests, I agree.